ui/src/common/PWA.js

```diff
@@ -0,0 +1,21 @@
+import useCurrentTheme from '../themes/useCurrentTheme'
+
+const ChangeColor = () => {
+  const theme = useCurrentTheme()
+  const query = document.querySelector("meta[name='theme-color']")
+
+  try {
+    const color =
```

Comment:
Shouldn't this logic be inside a `useEffect` hook, to avoid it running multiple times? Maybe it could be just a function, called from a `useEffect` hook inside the `Admin` component

Reply:
Thanks for your review; I have removed the if loop and used the `useEffect` hook.